File: 250318/등장횟수 확인/check-number-of-appearances.cpp
#include <iostream>
using namespace std;

int main()
{
    int count = 0;
    for(int i = 0; i< 5; i++) {
        int n = 0;
        cin >> n;
        if(n % 2 == 0) {
            count++;
        }
    }
    cout << count;
    return 0;
}

File: 250318/등장횟수 확인/check-number-of-appearances.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int count = 0;
        for(int i = 0; i <5;i++) {
            int n = sc.nextInt();
            if(n % 2 == 0) {
                count++;
            }
        }
        System.out.println(count);
        sc.close();
    }
}